src/LSST/simulator/SimPowerSubsystem.cpp

```diff
@@ -0,0 +1,133 @@
+/*
+ *  This file is part of LSST M2 support system package.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <http://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "simulator/SimPowerSubsystem.h"
+
+// System headers
+
+// Project headers
+#include "util/Bug.h"
+#include "util/Log.h"
+
+using namespace std;
+
+namespace LSST {
+namespace m2cellcpp {
+namespace simulator {
+
+SimPowerSubsystem::SimPowerSubsystem(control::PowerSubsystemConfig::SystemType systemType,
+        control::OutputPortBits::Ptr const& outputPort, int powerOnBitPos, int breakerResetPos,
+        control::InputPortBits::Ptr const& inputPort,  std::vector<int> const& breakerBitPositions)
+        : _systemType(systemType) ,_outputPort(outputPort),  _powerOnBitPos(powerOnBitPos), _breakerResetPos(breakerResetPos),
+          _inputPort(inputPort), _breakerBitPositions(breakerBitPositions) {
+    _setup();
+}
+
+void SimPowerSubsystem::_setup() {
+    control::PowerSubsystemConfig psc(_systemType);
+
+    _voltageNominal = psc.getNominalVoltage();
+
+     /// Once powered on, voltage should reach an acceptable level
+     /// before outputOnMaxDelay() time has past. See PowerSubsystemCommonConfig.vi
+     _voltageChangeRateOn = (_voltageNominal/psc.outputOnMaxDelay())*1.3;
```

Comment:
Please help to put `1.3` value into a `const` named variable to clarify the detail. Or you could explain the value of `1.3` in the comment instead.